### PR TITLE
Tb 20231810 assignment1

### PR DESCRIPTION
**Pull request by BG-Tia**
I would like to add this to main.

### CODE REVIEW

File: force-app/main/default/objects/Task__c/fields/Journey_del__c.field-meta.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            <valueField>$User.Id</valueField>
+        </filterItems>
+        <isOptional>false</isOptional>
+    </lookupFilter>
+    <referenceTo>Journey__c</referenceTo>
+    <relationshipLabel>Tasks</relationshipLabel>
+    <relationshipName>Tasks_del</relationshipName>
+    <required>false</required>
+    <trackTrending>false</trackTrending>
+    <type>Lookup</type>
```
Comment (UKJWL) [inline force-app/main/default/objects/Task__c/fields/Journey_del__c.field-meta.xml:21]: You're correct in assuming there is a relationship between the two objects. What happens when the related journey is deleted?

File: force-app/main/default/objects/Task__c/fields/Journey_del__c.field-meta.xml
[the remark this answers is not in the document]
Comment (UKJWL) [reply force-app/main/default/objects/Task__c/fields/Journey_del__c.field-meta.xml:1]: Does it make sense for a task to share both a template and a journey?

**Review by UKJWL — CHANGES_REQUESTED** [review]
Overall really solid first go at it. I like that you've thought about the context in which things are used, like creating an app. 

Descriptions are great, really clear to understand what the purpose is. 

The model itself has a bit of an issue, it's not major and it's probably a hurdle that most if not all the graduates don't spot first time. I know I didn't anyway. 

Here's some clues as to what you may be missing:

- You might want to think about how the relationships work in your implementation in some more _**detail**_.  Do the relationships you've chosen fit the purpose of the requirements? Have a look into the Salesforce relationship properties if you're unsure.
- 
- You've also got something in here that does serve the purpose as intended but it's merged together. Have another read of the requirements and let me know what you think it could be.

Overall really good start, nice one.

File: force-app/main/default/objects/Template__c/fields/Task__c.field-meta.xml
[the remark this answers is not in the document]
Comment (BG-Tia) [reply force-app/main/default/objects/Template__c/fields/Task__c.field-meta.xml:1]: Changed this~ template tasks no longer deleted on journey task deletion.

File: force-app/main/default/objects/Task__c/fields/Journey_del__c.field-meta.xml
[the remark this answers is not in the document]
Comment (BG-Tia) [reply force-app/main/default/objects/Task__c/fields/Journey_del__c.field-meta.xml:1]: Realised error here! Changed through task object and created 'Template Task' ~ a separate task object to make template tasks independent from journey.

File: force-app/main/default/objects/Task__c/Task__c.object-meta.xml
Comment (UKJWL) [inline force-app/main/default/objects/Task__c/Task__c.object-meta.xml:1]: Delete this pleas.

File: force-app/main/default/layouts/Task__c-Task Layout.layout-meta.xml
Comment (UKJWL) [inline force-app/main/default/layouts/Task__c-Task Layout.layout-meta.xml:1]: Delete this please if it doesn't automatically after you delete the object.

**Pull request merged**